colin/core/ruleset/ruleset.py

```diff
@@ -112,20 +113,18 @@ def get_checks(self, target_type, tags=None):
         return result
 
 
-def get_checks_path(checkpath=None):
+def get_checks_paths(checks_paths=None):
     """
     Get path to checks.
 
-    :return: str (absolute path of directory with checks)
+    :param checks_paths: list of str, directories where the checks are present
+    :return: list of str (absolute path of directory with checks)
     """
-    if checkpath is not None:
-        out_path = checkpath
-    elif os.environ.get(COLIN_CHECKS_PATH):
```

Comment:
it reverts my usecases for next fmf_scheduler, but it is okay, because I'll forward there this pathes from top level, original PR used env var directly there.
I'm fine with that, just I have to ensure to improve it in my PR :-)

Reply:
my point here is that when you are calling colin from API you can directly select the paths via `checks_paths`. Doing it via an env var would be really cumbersome